pkg/asset/machines/ovirt/machines.go

```diff
@@ -9,10 +9,9 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 
+	ovirtprovider "github.com/openshift/cluster-api-provider-ovirt/pkg/apis/ovirtprovider/v1beta1"
```

Comment:
Move this import into the previous section, with other non-installer imports.

Reply:
Done